### ISSUE TITLE
Sidebar doesn't load on remote connection first time

### ISSUE THREAD

**Issue by michaelmarconi** [elchininet/custom-sidebar#183]
Hi!

I've installed and customised the sidebar which works beautifully on my local network.  When I connect remotely via the Home Assistant Cloud however, the companion app renders the old sidebar.  Sometimes I can get the custom sidebar to load if I pull down to refresh the app but it's inconsistent, so possibly a caching issue.  

Switching on WiFi on my phone for local connection immediately causes the custom sidebar to appear.

Anyone else experiencing that?

Thanks!
Michael

**Comment by elchininet** [comment]
Hi @michaelmarconi,
Most probably a caching issue like [this one](https://github.com/elchininet/custom-sidebar/issues/131) or [this one](https://github.com/elchininet/custom-sidebar/issues/80#issue-2222795652). I access my Home Assistant instance remotely from the companion App and sometimes if I make a change the cache should be cleaned properly.

Are you on iOS or in Android? If you are on iOS just go to the sidebar, search for app configuration > debugging > reset frontend cache. If you are in Android, unfortunately there is no a native button to clean the Webview cache. Follow [these instructions](https://github.com/elchininet/custom-sidebar/issues/130#issuecomment-2327025323) to do so.

Try it out and let me know.

Regards

**Comment by elchininet** [comment]
Hi @michaelmarconi,
Did you try my recommendations?

**Comment by michaelmarconi** [comment]
Hi Ivan

I did but none of them helped.  I had noticed that my connection via Nabu Casa had become incredibly slow the last few days, so I decided to log out of that, restart my Home Assistant VM and log back in again and voila, all is working now.  

I’m not really sure what the cause of the problem was but I suspect there was some kind of connectivity or caching issue that was nothing to do with the frontend cache. 

In any case, thanks for your quick assistance and apologies for the delayed response!

Kind regards,
Michael
